pages/participantinfo.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+    Button,
+    Box,
+    Center,
+    Text,
+    Input,
+    FormControl,
+    FormLabel,
+    Progress,
+    Stack,
+    HStack,
+    VStack,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+/**
+ * This is the page that a user will use to enter the participants personal information
+ * onto the SDC platform as a parent of volunteer
+ */
+export default function Participantinfo(): JSX.Element {
```

Comment:
```suggestion
export default function ParticipantInfo(): JSX.Element {
```

Reply:
Will the title/header of the form be in this file as well? @mahadzaryab1 @sohak22

Reply:
yea it will be, I was just having difficulty figuring out why when I add content to the bottom of the page everything above it moves off the page. Going to work it through with @mahadzaryab1

Reply:
see https://github.com/uwblueprint/social-diversity-for-children/pull/43#discussion_r675061244